src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -220,6 +225,30 @@ public virtual async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAc
                 var tasks = tasksResponse?.Tasks.Where(t => t.ItemsDueCount > 0 && t.Type != "AgreementToSign").ToList() ?? new List<AccountTask>();
                 var showWizard = userResponse.User.ShowWizard && userRoleResponse.UserRole == Role.Owner;
 
+
+                var apprenticeshipsCount = 0;
+                IEnumerable<GetApprenticeshipsResponse.ApprenticeshipDetailsResponse> apprenticeshipResponse = await GetApprenticeshipResponse(accountResponse);

```

Comment:
> Can we move this out to a mediatr query and then the encoding service can be injected in there rather than being yet another dependency added to this orchestrator.

moved out to mediatR Handler